applications/minotari_miner/src/run_miner.rs

```diff
@@ -455,8 +457,10 @@ async fn get_new_block_base_node(
 async fn get_new_block_p2pool_node(
     sha_p2pool_client: &mut ShaP2PoolGrpcClient,
 ) -> Result<GetNewBlockResponse, MinerError> {
+    let mut pow_algo = PowAlgo::default();
```

Comment:
Why do this?

Reply:
why not just declare immediately with he correct value

Reply:
because it is simpler to do this as I did not see any method or something that would use `PowAlgos` type to have the right pow algo.

Reply:
Feels like there is a way, but this is fine for now

Reply:
Actually I did not find any better yet, since there is no method for that to initialize a `PowAlgo` from `PowAlgos` enum (it is the enum which contains `i32` value for `PowAlgo`), so not sure.